doc/developer/workflow.rst

```diff
@@ -91,18 +91,53 @@ documentation and packaging systems will be updated to reflect the next
 possible release name to allow for easy distinguishing.
 
 After one month the development branch will be renamed to
-``stable/MAJOR.MINOR``.  This process is not held up unless a crash or security
-issue has been found and needs to be addressed. Issues being fixed will not
-cause a delay.
+``stable/MAJOR.MINOR``.  The branche is a stable branch. This process is not
+held up unless a crash or security issue has been found and needs to
+be addressed. Issues being fixed will not cause a delay.
 
 Bugfix releases are made as needed at 1 month intervals until the next
-``MAJOR.MINOR`` relese branch is pulled. Depending on the severity of the bugs,
+``MAJOR.MINOR`` release branch is pulled. Depending on the severity of the bugs,
 bugfix releases may occur sooner.
 
 Bugfixes are applied to the two most recent releases. Security fixes are
 backported to all releases less than or equal to one year old. Security fixes
 may also be backported to older releases depending on severity.
 
+Long term support branches ( LTS support)
+-----------------------------------------
+
+Previous definition of releases prevents long term support of previous releases.
+For instance, bug fixes are not applied if the stable branch is too old. Also,
+it is not planned to apply backporting of features requests on that branch.
+
+Because the FRR users have a need to backport feature requests, or they want
+more bugfixes after the stable branch becomes too old, there is a need to
+provide support on a long term basis on that stable branch. If that support is
+applied on that stable branch, then the branch is a maintenance branch.
+
+Having a maintenance branch requires extra-work and requires one person to be
+in charge of that maintenance branch for a certain amount of time. This
+requirement is a self nomination from a person that proposes to handle the
+maintenance of that branch. The work can be handled by two people. In all cases,
+there must be at least one person that is in charge of the maintenance.
+It is important to note that if there is a disengagement during the maintenance
+time process, it will then not be possible to continue maintenance. FRR proposes
+that the nomination process for maintenance will have to come from people which
+are already present in the maintainer list.
+
+The maintenance branch duties are the following ones:
+
+- organise meetings on a (bi-)weekly or monthly basis, the handling of issues
+  and pull requested relative to that branch.
+
+- decide with the participants or the maintainers list the big changes on the
+  maintenance branch. For instance, the decision or not to perform a
+  backporting of a feature request.
```

Comment:
how about replacing the above with:
- Follow normal community process for deciding which code is accepted into the    maintenance branch.

Reply:
ok

Reply:
however, I kept the example case of (backporting or feature request) since I want to undeline that contrary to other workflow process, here it will be possible to add features requests. or doing backporting.